pages/index.tsx

```diff
@@ -1,51 +1,87 @@
+  Grid,
+  Heading,
+  HStack,
+  Link,
+  Stack,
+  Text,
+} from '@chakra-ui/react'
+import {
+  CreateListAlert,
+  CreateListDrawer,
   PaginationSection,
+  ResourceCard,
   SearchBar,
-  ServicesListItem,
   SortMenu,
 } from 'components'
-import { PaginationProvider, SortProvider, useAllServicesLists } from 'hooks'
+import {
+  CreateListProvider,
+  PaginationProvider,
+  SortProvider,
+  useCreateList,
+} from 'hooks'
+import { Resource } from 'models'
 import { NextPage } from 'next'
 import Head from 'next/head'
+import { useState } from 'react'
 import Select from 'react-select'
 
-export const HomePage: NextPage = () => {
+export const CreateListPage: NextPage = () => {
+  const createListHandler = useCreateList()
   const {
-    visibleServicesLists,
-    numServicesLists,
+    visibleServices,
+    numServices,
     setSearchQuery,
     taxonomyOptions,
     setTaxonomyFilters,
-    sortHandler,
     paginationHandler,
-  } = useAllServicesLists()
+    sortHandler,
+    onAlertOpen,
+    onDrawerOpen,
+  } = createListHandler
   const { paginatedData } = paginationHandler
 
   const sortFieldsTextToVal = { Name: 'name', Description: 'description' }
 
+  const [selectedResource, setSelectedResource] = useState<Resource>()
+  const saveResource = (resource: Resource) => {
+    setSelectedResource(resource)
+    onAlertOpen()
+  }
+
   return (
-    <Stack spacing="32px" p={{ base: '48px', md: '64px' }}>
+    <>
       <Head>
-        <title>Resource Lists</title>
+        <title>{'Create A List'}</title>
         <meta
           name="description"
-          content={
-            'A directory of lists of resources available for New Yorkers.'
-          }
+          content={'Create a customized list of resources that you can share.'}
         />
         <meta name="image" content="/manyc_logo.png" />
         <link rel="icon" href="/icon.ico" />
       </Head>
-      <Stack spacing="16px">
-        <Heading fontSize={{ base: '32px', md: '48px' }}>
-          Resource lists
+      <Box px="112px" py="96px">
+        <Text fontWeight="semibold" color="Primary.600" mb="12px">
+          Community Resources
+        </Text>
+        <Heading as="h1" fontSize="5xl" mb="24px">
+          Resource Hub
         </Heading>
-        <Text fontSize={{ base: '16px', md: '24px' }}>
-          Where you can view curated lists of resources or create your own!
+        <Text maxW="50%">
+          Our community-sourced, volunteer-curated library is a growing
+          collection of the many resources available to New Yorkers. Mutual Aid
+          NYC is committed to building a comprehensive list of high-quality
+          resources—check back frequently, as new resources are added every day!
+          <br /> <br />
+          If you can’t find what you need, search the map and borough-specific
+          lists in the Groups Directory to find groups that can help.
         </Text>
-      </Stack>
-      <Stack spacing="16px">
+      </Box>
```

Comment:
Most of the diffs in this file are because what was the `create-list.tsx` page is now the `index.tsx` page. The only real changes here are adding some sections with copy above and below the grid of resource cards.